Move-Mentor/move-mentor-front-end
# Fix footer add tests

Footer now stays at the bottom of the page regardless of the page content.

Wrote a test to check if the nav bar renders and will call either the header-teacher class (which has teacher header colour) or the header-student class (which has the student header colour).


**Steps to Test:**

- Git checkout fit-footer-add-tests
- Open a terminal and npm install (to make sure the test packages from main are installed)
- npm run test (might have to press a to select all tests)
- Watch for any errors
- Control + C to stop the test suite
